gengen/util/gengen.go

```diff
@@ -292,6 +292,19 @@ func setupMiners(st state.Tree, sm vm.StorageMap, keys []*types.KeyInfo, miners
 				return nil, err
 			}
 		}
+		if m.NumCommittedSectors > 0 {
```

Comment:
Because power is now added during `submitPoSt` this is needed for setting power in the genesis block.  Again let me know if this bootstrapping solution is flawed.

Reply:
This works for me.